internal/check/check.go

```diff
@@ -423,3 +423,32 @@ func NotPresent[T any, QT ygnmi.SingletonQuery[T]](query QT) Validator {
 		return nil
 	})
 }
+
+// UnOrderedEqual is to compare slices in un ordered way.
+func UnOrderedEqual[T any, QT ygnmi.SingletonQuery[T]](query QT, want T) Validator {
+	return Predicate(query, fmt.Sprintf("want %#v", want), func(got T) bool {
+
+		v1 := reflect.ValueOf(got)
```

Comment:
Rather than rolling our own comparison, let's just use the cmp library with the cmp.SortSlices option.

https://pkg.go.dev/github.com/google/go-cmp/cmp#Equal
https://pkg.go.dev/github.com/google/go-cmp/cmp/cmpopts#SortSlices

Reply:
updated, thank you